docs/source/Compiler.md

```diff
@@ -128,5 +128,8 @@ Additional options:
 -   `--reflect-types` : Add minimal type reflection to code generation.
 -   `--reflect-names` : Add minimal type/name reflection.
 
+-   `--force-defaults` : Emit default values for fields which are not present
```

Comment:
As you said, this is not for fields that are not present, merely for default values. maybe "emit default values in binary from json" as a more precise description.